english/dream/swim/swim.md

```diff
@@ -11,3 +11,15 @@ A pain shoots up your legs. You glance down to where your feet should be only to
 flipper wrapped tightly in the ends of licorice whips.
 
 You are trapped. Your dream is going horribly wrong. You have been ensnared by... 
+
+... Ursula the sea witch!
+
+She is enormous, with a mouth large enough to eat you in three bites! Her licorice-whip hair is 
+wrapped around your flipper, and she begins to pull you towards her, reeling in the licorice like 
+fishing line.
+
+What to do!?
+
+[Attempt to chew through the licorice lines ensnaring you.](/english/dream/swim/swim-trap/chew.md)
+[Pinch yourself to see if this is a dream.](./is-this-a-dream/is-this-a-dream.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.